### ISSUE TITLE
Where I can find prepared align lfw dataset?

### ISSUE THREAD

**Issue by abdikaiym01** [leondgarse/Keras_insightface#51]
Guys, could you get me a link to the dataset, preferably in .bin format like the author shows in the readme file. 
Thank you in advance)

**Comment by leondgarse** [comment]
It's contained in every training datasets [insightface/recognition/_datasets_](https://github.com/deepinsight/insightface/tree/master/recognition/_datasets_). You can just ask the author. :rofl:
I can also updated them here if you really need it this way.

**Comment by abdikaiym01** [comment]
I mean lfw dataset.

**Comment by leondgarse** [comment]
You mean the test set `lfw.bin`, right? Like `cfp-fp.bin` and `agedb-30.bin`, they are all contained in the training set. If you mean the `LFW` training dataset, I don't have it either...

**Comment by abdikaiym01** [comment]
Thank you, I found the dataset lfw.bin
